lib/wallet/src/Cardano/Wallet/CoinSelection.hs

```diff
@@ -384,6 +385,27 @@ toExternalSelectionSkeleton Internal.SelectionSkeleton {..} =
 -- Selections
 --------------------------------------------------------------------------------
 
+-- | Represents a unbalanced selection.
+--
+data PreSelection = PreSelection
```

Comment:
Could this be moved? E.g. to alongside `TransactionCtx`?

Reply:
yeah, sounds like good idea ...wonder only if it will not major ripple effects and affect other calls (other than constructTransaction)